### ISSUE TITLE
CPU占用异常

### ISSUE THREAD

**Issue by ziyue7575** [akof1314/Wiz.Editor.md#211]
保存的时候,内存占用不断升高,直到崩溃.
开始的时候只是十几,但是ctrl+S保存几下之后,CPU就爆表了.我试着直接使用为知笔记的默认编辑器或者typora外部编辑器,没有问题,用插件就会异常.
卸载插件重新安装还是没有解决
![image](https://user-images.githubusercontent.com/40845658/90382430-52ba5d80-e0b1-11ea-9123-1b1f43041e9c.png)
![image](https://user-images.githubusercontent.com/40845658/90384991-040ec280-e0b5-11ea-8fe3-e9c8ee5f7d2f.png)

**Comment by akof1314** [comment]
为知和插件都是最新版本吗

**Comment by ziyue7575** [comment]
> 为知和插件都是最新版本吗

是的,我昨天晚上又重新安装了为知笔记之后,发现若是修改几个字符就保存,CPU会占用大约15%,几秒钟就降下来了,但是,我复制了一小端话:
>主进程并不处理网络请求，主要负责调度工作进程，也就是图示的三项：加载配置、启动工作进程及非停升级。所以，nginx启动以后，查看操作系统的进程列表，我们就能看到至少有两个nginx进程。

粘贴一次保存一次,CPU直线上升,4、5次CPU就90%,并且CPU不降了.

**Comment by cloris-cc** [comment]
我也遇到这个问题了。当我使用一段时间CPU占用就变得特别高。（未升级前并无此情况）

**Comment by akof1314** [comment]
我再根据提供的文本检查下

**Comment by Lyn6969** [comment]
我也出现了这个情况，CPU占用突然飙升
